tests/filecheck/transforms/scf-parallel-loop-tiling-partial.mlir

```diff
@@ -54,9 +54,9 @@ func.func @tile_partial() {
 // CHECK-FIRST-NEXT:     ^{{.*}}(%{{.*}} : index, %{{.*}} : index, %{{.*}} : index):
 // CHECK-FIRST-NEXT:       %{{.*}} = arith.addi %{{.*}}, %{{.*}} : index
 // CHECK-FIRST-NEXT:       %{{.*}} = arith.addi %{{.*}}, %{{.*}} : index
-// CHECK-FIRST-NEXT:       scf.yield
+// CHECK-FIRST-NEXT:       "scf.reduce"() : () -> ()
```

Comment:
this is definitely a nit, but it's a bit of a shame to add so many ops in generic syntax when the custom syntax is not very involved. Do you know if the custom syntax has changed from our version? I could quickly open a PR with the custom syntax with our current MLIR, to avoid this micro regression

Reply:
`scf.reduce` has a non-trivial custom syntax I believe, and it'll be completely different to the mlir used for xdsl main

Reply:
Ah, ok, it looks fairly straightforward in the new docs, would it make sense to copy that as part of this PR? I think if it works without too much effort then it would probably be beneficial

Reply:
You're right, it doesn't look as bad as I thought (`scf.parallel` on the othre hand is bad)